.github/workflows/acceptance.yml

```diff
@@ -153,34 +153,14 @@ jobs:
       TEST_GROUP_STRATEGY: roundrobin
 
     steps:
-      - uses: actions/checkout@v2
-        name: Checkout sentry
-
-      - name: Set python version output
-        id: python-version
-        run: |
-          echo "::set-output name=python-version::$(cat .python-version)"
-
-      # Until GH composite actions can use `uses`, we need to setup python here
-      - uses: actions/setup-python@v2
-        with:
-          python-version: ${{ steps.python-version.outputs.python-version }}
-
-      - name: Setup pip
-        uses: ./.github/actions/setup-pip
-        id: pip
-
-      - name: pip cache
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.pip.outputs.pip-cache-dir }}
-          key: |
-            ${{ runner.os }}-py${{ steps.python-version.outputs.python-version }}-pip${{ steps.pip.outputs.pip-version }}-${{ secrets.PIP_CACHE_VERSION }}-${{ hashFiles('requirements-*.txt', '!requirements-pre-commit.txt') }}
```

Comment:
How come this doesn't need `uses: ./.github/actions/setup-python` anywhere?

Reply:
Same question for a few below

Reply:
These use `./.github/actions/setup-sentry` which uses `setup-python`

Reply:
Where's the `use:` though?

Reply:
https://github.com/getsentry/sentry/pull/28281/files#diff-20178bfee541d27c8c2296d927b63f9c6fec922b9ead4965f2e35958204baebbR160

Reply:
and https://github.com/getsentry/sentry/pull/28281/files#diff-1ea2e237b0bccd6eb6f5b3b3dee00fede80d82d4e56d97a63acd9a7c37444e29R93-R95